src/game/Box.js

```diff
@@ -25,16 +25,17 @@ export default class Box extends Component {
         const pieceImageArray = initImagesRequire();
 
         return (
-            <label>
-                {label}
+            <View testID="gridbox">
```

Comment:
`data-testid`

Reply:
"testID" is the official way to test apps for e2e (https://facebook.github.io/react-native/docs/view.html#testid)
It's used in Detox as well

Reply:
Agreed but that's not compatible with `react-testing-library` which we use here

Reply:
So we're going to extend `react-testing-library` locally